### PR TITLE
Update DIA SDK filepaths and DLL names

### PR DESCRIPTION
**Pull request by mavaddat**
I have updated the filepaths and updated the DLL names for the Debug Interface Access (DIA) SDK docs with examples from Visual Studio 2022. These are from my own system
I calculated the paths using the following helper script that I wrote in PowerShell:

```pwsh
Get-ChildItem -Path "$env:ProgramFiles\Microsoft Visual Studio\" -Filter 'msdia*dll' -Recurse | ForEach-Object {
    $diaPath = $_.FullName
    foreach ($vsVar in Get-ChildItem -Path Env:\ | Where-Object -FilterScript { $_.Name -match 'visual' -or $_.Value -match 'visual' -and (Test-Path -Path $_.Value) } | ForEach-Object {
            [pscustomobject]@{
                Length = $_.Value.Length
                Name   = $_.Name
                Value  = $_.Value
            }
        } | Sort-Object -Descending -Property Length )
    {
        $diaPath = $diaPath -replace ([regex]::Escape(($vsVar.Value))), ('%' + $vsVar.Name + '%\')
    }
    $diaPath | Write-Output
} | Sort-Object -Unique
```

### CODE REVIEW

**Comment by Court72** [comment]
@Mikejo5000

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/debugger/debug-interface-access/getting-started-debug-interface-access-sdk.md [hunk 1]
```diff
@@ -19,25 +19,31 @@ ms.workload:
...
 
 - C++ programming language
 
 - COM programming
 
 - Visual Studio integrated development environment (IDE) for compiling the samples
 
-  The DIA SDK is normally installed with Visual Studio and its default location is *[drive]*\Program Files\Microsoft Visual Studio 9.0\DIA SDK. As part of the installation, the msdia90.dll, which implements the DIA SDK, is automatically registered so all that you need to do to use it is to include `dia2.h` in your program and link to `diaguids.lib`.
+  The DIA SDK is normally installed with Visual Studio and its default location is `%VSINSTALLDIR%\DIA SDK\bin` with associated subdirectories for additional processor architectures.     For example, 
+  - `C:\Program Files\Microsoft Visual Studio\2022\Enterprise\DIA SDK\bin\msdia140.dll` (default)
```
Comment (Mikejo5000) [inline docs/debugger/debug-interface-access/getting-started-debug-interface-access-sdk.md:33]:
```suggestion
  - `C:\Program Files\Microsoft Visual Studio\2022\Enterprise\DIA SDK\bin\msdia140.dll` (x86)
```

File: docs/debugger/debug-interface-access/getting-started-debug-interface-access-sdk.md [hunk 2]
```diff
@@ -19,25 +19,31 @@ ms.workload:
...
+  - `C:\Program Files\Microsoft Visual Studio\2022\Enterprise\DIA SDK\bin\amd64\msdia140.dll` (amd64)
+  - `C:\Program Files\Microsoft Visual Studio\2022\Enterprise\DIA SDK\bin\arm\msdia140.dll` (arm)
+  - `C:\Program Files\Microsoft Visual Studio\2022\Enterprise\DIA SDK\bin\arm64\msdia140.dll` (arm64)
+ 
+  As part of the installation, the `msdia140.dll`, which implements the DIA SDK, is automatically registered so all that you need to do to use it is to include `dia2.h` in your program and link to `diaguids.lib`.
```
Comment (Mikejo5000) [inline docs/debugger/debug-interface-access/getting-started-debug-interface-access-sdk.md:38]:
```suggestion
  As part of the installation, the `msdia140.dll`, which implements the DIA SDK, is automatically registered so all that you need to do to use it is to include `dia2.h` in your program and link to `diaguids.lib`.
```

Comment (Mikejo5000) [reply docs/debugger/debug-interface-access/getting-started-debug-interface-access-sdk.md:38]: @mavaddat Apparently, this is no longer true so can be removed

**Review by Mikejo5000 — CHANGES_REQUESTED** [review]
@mavaddat I put in a few GitHub suggestions requested by one of the devs. Thanks for the contribution! Looks good. Please use the **Commit suggestion** buttons to add the suggestions.

File: docs/debugger/debug-interface-access/getting-started-debug-interface-access-sdk.md [hunk 3]
```diff
@@ -19,25 +19,30 @@ ms.workload:
...
 
 - C++ programming language
 
 - COM programming
 
 - Visual Studio integrated development environment (IDE) for compiling the samples
 
-  The DIA SDK is normally installed with Visual Studio and its default location is *[drive]*\Program Files\Microsoft Visual Studio 9.0\DIA SDK. As part of the installation, the msdia90.dll, which implements the DIA SDK, is automatically registered so all that you need to do to use it is to include `dia2.h` in your program and link to `diaguids.lib`.
+  The DIA SDK is normally installed with Visual Studio and its default location is `%VSINSTALLDIR%\DIA SDK\bin` with associated subdirectories for additional processor architectures.     For example, 
+  - `C:\Program Files\Microsoft Visual Studio\2022\Enterprise\DIA SDK\bin\msdia140.dll` (default)
```
Comment (Mikejo5000) [inline docs/debugger/debug-interface-access/getting-started-debug-interface-access-sdk.md:33]:
```suggestion
  - `C:\Program Files\Microsoft Visual Studio\2022\Enterprise\DIA SDK\bin\msdia140.dll` (x86)
```

**Review by Mikejo5000 — APPROVED** [review]

**Comment by Mikejo5000** [comment]
#sign-off Thanks, @mavaddat !

**Pull request merged**